packages/insomnia-app/app/plugins/install.js

```diff
@@ -170,6 +172,37 @@ async function _installPluginToTmpDir(lookupName: string): Promise<{ tmpDir: str
   });
 }
 
+function isFalsePositive(stderr) {
+  // Split on line breaks and remove falsy values (null, undefined, 0, -0, NaN, "", false)
+  const arr = stderr.split(/\r?\n/).filter(e => e);
+  // Retrieve all matching deprecated dependency warning
+  const warnings = arr.filter(e => isDeprecatedDependencies(e));
```

Comment:
These filters can be combined to reduce the number of times we iterate through the array

```suggestion
  const arr = stderr.split(/\r?\n/);
  // Retrieve all matching deprecated dependency warning
  const warnings = arr.filter(e => e && isDeprecatedDependencies(e));
```

Reply:
I cannot do that otherwise the algorithm mismatch the array length and the function returns `false` when it shouldn't.
- **If i do not apply** the last filter, `arr` will contain [msg, null] and `warnings` [msg] and it's mismatched.
- **If i do apply** the last filter, arr will contain [msg] and warnings [msg] and it's a match.

Reply:
Ah, that makes sense! 👍